### ISSUE TITLE
Cleanup old Fork.exe_Url_... folders

### ISSUE THREAD

**Issue by MAG-CK** [ForkIssues/TrackerWin#1775]
I don't know why these folders are present, I think for an update of Fork. But why are so many folders alive after an update?

![image](https://user-images.githubusercontent.com/120392576/215740343-49d934a7-87a5-4c4a-846e-bab445305e1a.png)

I can see the "1.72.3.0" version of Fork in one of these folders. Is there no cleanup?

**Comment by DanPristupov** [comment]
No, there's no cleanup for those folders. They are created by a library and not directly by Fork.

**Comment by DanPristupov** [comment]
I'll probably make Fork remove them on update, just to keep the installation directory clean.

**Comment by MAG-CK** [comment]
> They are created by a library and not directly by Fork.

By Squirrel?

> I'll probably make Fork remove them on update, just to keep the installation directory clean.

That sounds good, thanks

**Comment by MAG-CK** [comment]
No chance to delete the folders? I got a lot more with the last updates.
![image](https://github.com/fork-dev/TrackerWin/assets/120392576/f63b63a9-ff1e-40cc-83dd-deafdcf4e72a)

**Comment by DanPristupov** [comment]
> No chance to delete the folders?

You can delete them manually if they make a problem for you.

**Comment by jerone** [comment]
FYI. I've got 162 of those folders. Going back to 2019.
They accumulate only 320 KB, so I'm not worried.
But a cleanup would be nice.
